README.md

```diff
@@ -1,2 +1,8 @@
-# ci.docker.websphere-classic
-Dockerfiles for WebSphere Application Server classic
+# WebSphere Application Server Classic and Docker
+
+Under this directory you can find build scripts for Docker images (Dockerfiles) and Docker related utilities for WebSphere Application Server Classic. 
+
+* [WebSphere Application Server Classic Base](base)
```

Comment:
Perhaps structure these as:

* WebSphere Application Server Classic for Developers
* WebSphere Application Server Classic
** Base
** Network Deployment